src/textual/widgets/_tabbed_content.py

```diff
@@ -26,15 +27,46 @@
 class ContentTab(Tab):
     """A Tab with an associated content id."""
 
-    def __init__(self, label: Text, content_id: str, disabled: bool = False):
+    _PREFIX: Final[str] = "--content-tab-"
```

Comment:
Ah, I was just reminded of one last thing!

The users need to know what this prefix is if they want to write their own CSS that styles tabs, right?
That's why this whole conversation started.

Wouldn't it be a good idea if this were actually a public attribute of the class, or maybe even a public global?
This way, it's easier to document.

I'm not sure this is a good idea, so I'll just leave this as a comment and not as a requested change.

Reply:
I did consider this but the problem I see is that 99.9% of the time, when they want to use it, it'll be in CSS; where some public constant would be utterly useless.

Reply:
I think your statistic looks accurate. I only wondered that because of documentation. With a leading `_`, it won't show in the documentation.

Reply:
I would hope it doesn't.

Reply:
😆 then one of us misunderstood the other.
If the prefix isn't documented, how can the user know what to write in CSS?

Reply:
I'm not in a position to check properly right now, but I have a recollection of documenting it (hence the thing about not being able to build the docs last week).

Reply:
![image](https://github.com/Textualize/textual/assets/28237/9d719de7-4bbf-4daa-bf31-ce43a073d6e2)

Reply:
I saw that.
If/when we change the value of `_PREFIX`, that will need updating and it's likely we'll forget to do so.
If instead it linked to `ContentTabs.PREFIX` in the documentation, then updating the code would be enough.
That was my only concern.

Reply:
My view is it hides the detail in the wrong location; and it would need (or at least deserve) the example code anyway, leaving exactly the same problem.